south_migrations/0001_initial.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
```

Comment:
It is, but the pull request is showing the diff with all four diffs applied at the same time.